### DISCUSSION TITLE
Codespace terminal gets launched in wrong directory

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#26378] [Codespaces]
<p>I noticed that since today (or maybe yesterday) freshly created codespaces launch the terminal in the wrong directory. As you can see from attached screenshot, the codespace gets launched in the <code>workspace</code> directory and not in the directory that actually contains the cloned project. I am basically one level too high.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/a/fabca23fc987cedf8502c79e4b472aa453e6213f.png" data-download-href="/uploads/short-url/zM7vQptxG5qXVFNGLUGt3bJdOWz.png?dl=1" title="Screen Shot 2020-11-16 at 10.33.00 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181097396-add3f10b-a8cf-4aae-aeea-fccba603971d.png" alt="Screen Shot 2020-11-16 at 10.33.00 PM" data-base62-sha1="zM7vQptxG5qXVFNGLUGt3bJdOWz" width="690" height="368" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/a/fabca23fc987cedf8502c79e4b472aa453e6213f_2_690x368.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/a/fabca23fc987cedf8502c79e4b472aa453e6213f_2_1035x552.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/a/fabca23fc987cedf8502c79e4b472aa453e6213f_2_1380x736.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/a/fabca23fc987cedf8502c79e4b472aa453e6213f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-11-16 at 10.33.00 PM</span><span class="informations">2578×1378 186 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I can <code>cd</code> on the terminal and work on the project, but the issue is that being in the parent directory breaks some project configs, e.g. for ESLint. I am not sure when exactly this problem started but I think I can narrow it down to today or yesterday.</p>

**Comment by matthewisabel** [comment]
<p>Thanks for raising this. It’s something we’re tracking and plan to fix shortly.</p>

**Comment by maiertech** [comment]
<p>Is there an ETA for this? This issue seems somewhat random. Yesterday, some codespaces worked just fine, while with others I ran into this issue. Today every codespace I launch has this issue.</p>

**Comment by jkeech** [comment, marked as answer]
<p>The fix for this has been rolled out. Please let us know if you continue to see it. The issue was that the root path was being set incorrectly on resume. During creation, it was set to the repo root, but on resume, it was set one level higher. Now resume keeps it at the correct path. Both existing Codespaces and new Codespaces should be fixed now on the next resume.</p>
